.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY = &key=AIzaSyBsZof6jxUT9CDKHcp4QVQWOcB-95uDKxg
```

Comment:
You have uploaded your `.env` file, remove it

Reply:
yeah sure